src/internal/kopia/upload.go

```diff
@@ -262,6 +263,14 @@ func (cp *corsoProgress) CachedFile(fname string, size int64) {
 // during the upload process. This could be from reading a file or something
 // else.
 func (cp *corsoProgress) Error(relpath string, err error, isIgnored bool) {
+	// The malware case is an artifact of being unable to skip the item
+	// if we catch detection at a late enough stage in collection enumeration.
+	// This is our next point of error handling, where we can identify and
+	// skip over the case.
+	if clues.HasLabel(err, graph.LabelsMalware) {
```

Comment:
I'm not really comfortable with modifying the returned stats from kopia post-backup as it could lead to hard to debug problems in the future. I think we're kind of conflating the information kopia returns with the information we'd like to persist in Corso and display to users later on

Reply:
Right.  I was hoping less for a post-stat modification, and more for a way to tell kopia "this isn't an error, don't worry about it, skip the file and move on" in the first place.  But since the error source is embedded in the lazy reader, I'm having trouble finding a good way to do that.

Reply:
I don't think there is a way to do that and again, I don't think we really should be doing stuff like that. Kopia is really meant to deal with static file systems, so any error getting file data is a big issue in that context

Reply:
yes, I get that not handling this in kopia wrapper may mean we need to rethink how we handle errors in other layers of the stack. Adding that handling to kopia wrapper though seems like it leaks stuff from other layers into kopia which will complicate stuff down the road

Reply:
Totally with you.  Worth a check.  Hmm, I'll just have to find another way around the issue.